### PR TITLE
2.6

### PR DESCRIPTION
**Pull request by hasnain2808**
Description
Updates branch created from [mcohen:2.6](https://github.com/macohen/opensearch-learning-to-rank-base/tree/2.6) to build with OpenSearch 2.6

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/release-old.yml [hunk 1]
```diff
@@ -25,7 +23,7 @@ jobs:
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
     - name: Build with Gradle
-      run: ./gradlew -Dtests.security.manager=false build
+      run: ./gradlew -Dskip.integtests=true -Djava.security.manager=allow -Dbuild.snapshot=false
```
Comment (macohen) [inline .github/workflows/release-old.yml:26]: We'll want to ensure we can build snapshots and not skip integration tests, but I think this is an improvement that we can take for now. @hasnain2808 would you mind creating two new issues in opensearch-project/opensearch-learning-to-rank-base for this?

**Review by macohen — APPROVED** [review]

**Comment by macohen** [comment]
@hasnain2808, moving forward, please feel free to create your PRs against the opensearch-project/opensearch-learning-to-rank-base repo. You'll get more eyes on it that way. And thanks for this contribution!

**Comment by hasnain2808** [comment]
> @hasnain2808, moving forward, please feel free to create your PRs against the opensearch-project/opensearch-learning-to-rank-base repo. You'll get more eyes on it that way. And thanks for this contribution!

Hey @macohen 

Thanks for merging the PR!

Since you already had work done for 2.6 and 2.7 I based off my work for those versions on your prs

Moving ahead for other versions will base it of the appropriate main repo branches!

**Comment by macohen** [comment]
Are you open to working on a 2.11 release? I'll branch the 2.7 release now and you can check that out if you want.

**Comment by hasnain2808** [comment]
> Are you open to working on a 2.11 release? I'll branch the 2.7 release now and you can check that out if you want. 

Yes @macohen

**Comment by macohen** [comment]
OK. I just want to make sure we get more eyes on the 2.7 branch first.

**Pull request merged**